public/src/App.jsx

```diff
@@ -1,25 +1,28 @@
 import React from 'react';
-
+import { ThemeProvider } from 'styled-components';
 import {
   BrowserRouter as Router,
   Switch,
   Route,
 } from 'react-router-dom';
+import theme from './theme';
 import FormView from './views/FormView';
 import { ToastManager } from './components/ToastManager';
 import PageLayout from './components/PageLayout';
 
 const App = () => (
   <Router>
-    <PageLayout>
-      <ToastManager>
-        <Switch>
-          <Route path="/">
-            <FormView />
-          </Route>
-        </Switch>
-      </ToastManager>
-    </PageLayout>
+    <ThemeProvider theme={theme}>
```

Comment:
This is nit picky, but I generally define the theme & theme provider together.

```jsx
export const PottyTrackerTheme = (props) => <ThemeProvider theme={theme} {...props} />;
```

It's not common to have more than one use of the theme provider, but this way the correct pairing is done by one component. Additionally if you decide to do things like "dark mode" contexts and things like that, you can construct them in the correct way in one spot and update all usages.

Reply:
Fixed!